extensions/MYTAditya/CommentPlus.js

```diff
@@ -0,0 +1,383 @@
+                defaultValue: "comment",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterPlusPlus",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[INPUT] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.BOOLEAN,
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanPlusPlus",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "[INPUT] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterP6",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[COLOR] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanP6",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "[COLOR] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterP7",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "// [COMMENT] [COLOR]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanP7",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "// [COMMENT] [COLOR]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentCPlus",
```

Comment:
Why are there 2 of these?

Reply:
Oh it was actually I wanted to write `commentCPlusPlus` at that line